drivers/bridge/setup_ip_tables.go

```diff
@@ -149,7 +149,7 @@ func setIcc(bridgeIface string, iccEnable, insert bool) error {
 			iptables.Raw(append([]string{"-D", chain}, dropArgs...)...)
 
 			if !iptables.Exists(table, chain, acceptArgs...) {
-				if output, err := iptables.Raw(append([]string{"-A", chain}, acceptArgs...)...); err != nil {
+				if output, err := iptables.Raw(append([]string{"-I", chain}, acceptArgs...)...); err != nil {
```

Comment:
This was done through this PR in docker/docker just before libnetwork cutover https://github.com/docker/docker/pull/11526

But IMO there is a bug in that PR which is that only the DROP rule when `--icc=false` should be appended because you want to first look at the DOCKER chain to see if there are any rules for linked containers and if not then then fallback to the DROP rule at the end of the chain. But ACCEPT rule when `--icc=true` should be in the beginning and not at the end. So the PR was incorrect in moving the ACCEPT rule to the end.

This PR is fixing that so this chainge LGTM.